src/components/card.jsx

```diff
@@ -3,15 +3,61 @@ import PropTypes from 'prop-types';
 
 import '../styles/components/card.scss';
 
-const Card = ({ title, children }) => (
+const DEFAULT_BORDER_COLOR = '#e5e5e5';
```

Comment:
It would be best if the default was taken from the css so it can be set as a SASS variable (I believe the `style` tag overwrites the default so you should be able to leave it empty if you want to use the css style?)

Reply:
Done.